app/coffee/Features/User/UserController.coffee

```diff
@@ -44,6 +44,7 @@ module.exports = UserController =
 
 	updateUserSettings : (req, res)->
 		user_id = AuthenticationController.getLoggedInUserId(req)
+		usingExternalAuth = !!(settings.ldap? or settings.saml?)
```

Comment:
wouldn't this always result in a true or false?

usingExternalAuth = settings.ldap? or settings.saml?